lib/tpm2_util.h

```diff
@@ -93,6 +93,19 @@
         .rspAuths = array, \
     }
 
+/*
+ * This macro is useful as a wrapper around SAPI functions to automatically
+ * retry function calls when the RC is TPM_RC_RETRY.
+ */
+#define TSS2_RETRY_EXP(expression)                         \
+    ({                                                     \
+        TSS2_RC __result = 0;                              \
+        do {                                               \
+            __result = (expression);                       \
+        } while ((__result & 0x0000ffff) == TPM_RC_RETRY); \
```

Comment:
@flihp I'm surprised there isn't a mask for this in the TSS

Reply:
I've had a few things that I've wanted to work on and cleaning up the response code decoding is one of them. I don't even think the mask is necessary given the way TPM_RC_RETRY and the like are defined.